examples/testing/portal.py

```diff
@@ -0,0 +1,38 @@
+import asyncio
```

Comment:
Use `anyio` instead of asyncio here. This better highlights `anyio support.

Reply:
I'm struggling a bit with this one. One of my example includes how to spawn a background task during startup and check on it after the app has started. According to [anyio issue #457](https://github.com/agronholm/anyio/issues/457), this is not and will not be supported since there is no inherent monitor for the task; any exceptions are thrown into the void if nothing explicitly checks on it.

It sounds like a task scheduler would a better option but, for the sake of simplicity, that is not an option here. I could change the example but I was also trying to show to await something which is running on `Starlite`'s event loop.